curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -250,6 +250,13 @@ CURVEFS_ERROR FuseS3Client::FuseOpMkNod(fuse_req_t req, fuse_ino_t parent,
     return MakeNode(req, parent, name, mode, FsFileType::TYPE_S3, rdev, e);
 }
 
+CURVEFS_ERROR FuseS3Client::FuseOpUnlink(fuse_req_t req, fuse_ino_t parent,
```

Comment:
why move this function to fuse_s3_client.cpp？

Reply:
because we need fstype here